packages/modules/src/modules/banners/momentTemplate/MomentTemplateBanner.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { css } from '@emotion/react';
 import { neutral, space } from '@guardian/src-foundations';
-import { Container, Hide } from '@guardian/src-layout';
+import { Container } from '@guardian/src-layout';
```

Comment:
This was the last one `Hide`-ing in the moment template 🎉

Reply:
very satisfying :)